modules/ROOT/pages/prerequisites/index.adoc

```diff
@@ -93,6 +124,14 @@ When a client triggers a request to manage e.g. documents and versions (aka PROP
 
 NOTE: Bandwidth can be scaled by adding more gateways and distributing users between them because these components are stateless.
 
+== Virtualisation
+
+Depending on the usecase, you can run oCIS on:
+
+* Bare metal
```

Comment:
Does it really make sense to mention bare metal in the virtualization section, particularly since it's already covered above?

Reply:
Maybe:  No virtualisation, bare metal

Reply:
Better, but virtualization with z not s.